### PR TITLE
Refactor Harvest Gateway/tests

### PR DESCRIPTION
**Pull request by yusufsheiqh**

### CODE REVIEW

File: CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs [hunk 1]
```diff
@@ -15,22 +15,31 @@ public class HarvestGatewayTests
         private const string Token = "xxxx-xxxxxxxxx-xxxx";
         private const string HarvestAccountId = "123456";
         private const string UserAgent = "The Wolves";
-        private const string DeveloperRoles =
-            "Software Engineer, Senior Software Engineer, Senior Engineer, Lead Engineer, " +
-            "Delivery Manager, SRE, Consultant, Delivery Principal";
+        private const string DeveloperRoles = 
+            @"Software Engineer, Senior Software Engineer, Senior Engineer, Lead Engineer, 
+            Delivery Manager, SRE, Consultant, Delivery Principal";
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs]: Brilliant idea!

Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs]: Thank you.

**Comment by chubberlisk** [comment]
Looks good to me!

**Comment by yusufsheiqh** [comment]
> Looks good to me!

Thank you [2].

**Pull request merged**